NumLib/Assembler/SerialExecutor.h

```diff
@@ -76,6 +76,26 @@ struct SerialExecutor
         }
     }
 
+    /// Executes the given \c method on each element of the input \c container.
+    ///
+    /// This method is very similar to executeMemberDereferenced().
+    ///
+    /// \param container collection of objects having pointer semantics.
+    /// \param method    the method being called, i.e., a member function pointer
+    ///                  to a member function of the \c container's elements.
+    /// \param Args      types of further arguments passed on to the method
```

Comment:
warning: Parameter 'Args' not found in the declaration --> 'args'

Also in other functions in the SerialExecutor, but this one is new...